src/gh/components/DF_vizualization/metadata.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "DFVizualization",
+    "nickname": "DFViz",
+    "category": "diffCheck",
+    "subcategory": "Vizualization",
```

Comment:
typo: "VIsualization"

Reply:
Thnx! And it's also everywhere :p Hope i got them all